RBQFetchedResultsController/Source/Swift/Utilities.swift

```diff
@@ -59,7 +59,7 @@ extension Realm {
     
     :nodoc:
     */
-    internal class func toRLMConfiguration(configuration: Configuration) -> RLMRealmConfiguration {
+    internal class func toRLMConfiguration(_ configuration: Configuration) -> RLMRealmConfiguration {
```

Comment:
To support Realm v2+ with Realm Mobile Platform, I think this needs to be updated to include passing along the `syncConfiguration`:

```swift
if let syncConfig = configuration.syncConfiguration {
    let rlmSyncConfig = RLMSyncConfiguration(user: syncConfig.user, realmURL: syncConfig.realmURL)
}
```
and I think this too to add support for limiting the Realm to specific classes:
```swift
if let objectTypes = configuration.objectTypes {
    rlmConfiguration.objectClasses = objectTypes.map({ (type) -> AnyClass in
        return type as! AnyClass
    })
}
```